apps/explorer/src/app/routes/blocks/id/index.tsx

```diff
@@ -23,24 +29,29 @@ const Block = () => {
 
   return (
     <section>
-      <h1>BLOCK {block}</h1>
-      <Table>
-        <tr>
-          <td>Mined by</td>
-          <td>
-            <Link to={`/validators/${header.proposer_address}`}>
+      <RouteTitle>BLOCK {block}</RouteTitle>
+      <Table className="mb-28">
+        <TableRow modifier="bordered">
+          <TableHeader scope="row">Mined by</TableHeader>
+          <TableCell modifier="bordered">
+            <Link
+              className="text-vega-yellow"
+              to={`/validators/${header.proposer_address}`}
```

Comment:
chore (blocking): let's redirect to the all validators page until we build single valdiator pages

```suggestion
              to={"/validators"}
```